package-lock.json

```diff
@@ -52,6 +52,120 @@
         "node-addon-api": "^2.0.0"
       }
     },
+    "@grpc/grpc-js": {
```

Comment:
Honestly, Because we're developing on windows and deploying on linux, I've excluded this file from the docker file. So maybe we should also exclude this file in our gitignore

Reply:
addressed in eaca4fa